igvm_defs/src/lib.rs

```diff
@@ -597,13 +599,13 @@ pub struct IGVM_VHS_RELOCATABLE_REGION {
 /// is.
 ///
 /// # Architecture
-/// X64 only.
+/// X64 and ARM64.
 #[repr(C)]
 #[derive(Copy, Clone, Debug, AsBytes, FromBytes, FromZeroes, PartialEq, Eq)]
 pub struct IGVM_VHS_PAGE_TABLE_RELOCATION {
     /// Compatibility mask.
     pub compatibility_mask: u32,
-    /// VP Index for paging information, like CR3 and EFER.
+    /// VP Index for paging information, like the base page register.
```

Comment:
can we be specific what registers we're looking at for arm64 and x64?

Reply:
I added the current list.